src/promptflow/promptflow/_sdk/_submitter/run_submitter.py

```diff
@@ -96,16 +96,20 @@ def _submit_bulk_run(self, flow: Flow, run: Run, local_storage: LocalStorageOper
                 output_dir=local_storage.outputs_folder,
                 run_id=run_id,
             )
-
+            error_log = ""
             if batch_result.failed_lines > 0:
                 # Log warning message when there are failed line run in bulk run.
                 error_log = f"{batch_result.failed_lines} out of {batch_result.total_lines} runs failed in batch run."
-                if run.properties.get(FlowRunProperties.OUTPUT_PATH, None):
-                    error_log = (
-                        error_log
-                        + f" Please check out {run.properties[FlowRunProperties.OUTPUT_PATH]} for more details."
-                    )
-                logger.warning(error_log)
+            elif batch_result.error_summary.aggr_error_dict:
+                # log warning message when there are failed aggregation nodes in bulk run.
+                aggregation_nodes = list(batch_result.error_summary.aggr_error_dict.keys())
+                error_log = f"aggregation nodes {aggregation_nodes} failed in batch run."
```

Comment:
Batch runs may have both line failures and aggregation node failures, and if-if should probably be used instead of if-elif.

Reply:
Good catch!